apiserver/pkg/chat/chat_docs.go

```diff
@@ -210,13 +211,13 @@ func (cs *ChatServer) SummarizeConversationDoc(ctx context.Context, req Conversa
 		klog.V(5).Infof("Generate embeddings from file %s to vectorstore for conversation %s", doc.Filename, req.ConversationID)
 		errEmbedding = cs.GenerateSingleDocEmbeddings(ctx, req, doc, documents)
 		if errEmbedding != nil {
+			klog.V(1).ErrorS(errEmbedding, "ErrEmbedding", "document", doc.Filename, "conversation", "")
```

Comment:
```suggestion
			klog.V(1).ErrorS(errEmbedding, "ErrEmbedding", "document", doc.Filename, "conversation", req.ConversationID)
```
maybe you want this?